queries/highlights.scm

```diff
@@ -20,18 +23,30 @@
 "ternary" @keyword
 "optional" @keyword
 "actions" @keyword
+"default" @keyword
 "default_action" @keyword
 "return" @keyword
 "NoAction" @keyword
 "const" @keyword
 "meters" @keyword
 "counters" @keyword
-(direction) @keyword
-"bool" @type
-(bit_type) @type
+"_" @keyword
+"true" @constant
+"false" @constant
+(direction) @
```

Comment:
I don't think this is a valid highlighting directive. Helix does not accept it.

Reply:
@rcgoodfellow that's meant to be keyword, not sure what happened. apologies.